activation/activation.go

```diff
@@ -140,6 +145,8 @@ func (b *Builder) loop() {
 	}
 }
 
+// PublishActivationTx attempts to publish an atx for the given epoch ech, it returns an error if an atx cannot be created
```

Comment:
```suggestion
// PublishActivationTx attempts to publish an atx for the given epoch, it returns an error if an atx cannot be created
```
(this is not the argument name)